simpleviewinc/keg-cli
# Tap dependency task

## Goal
* Added tap dependencies task
  * Helper task to check a taps installed keg-hub repos and ensure they are up to date
  * Will auto-update a tap's `package.json` file 
  * Does **NOT** run any git commands

## Updates
* Added tap dependencies task
* Migrate more utils to cli-utils repo
* Fixed a number imports to point to cli-utils repo

## Testing
* Pull this pr => `keg cli && keg pr 58`
* Run the `dependencies` task for any linked taps
  * For example, using `tap-events-force`
    * Run command => `keg evf deps`
      *  Should print out that `@keg-hub/jsutils` is out of date
      * [Looks like this](https://prnt.sc/139jgpi)
    * Next run cmd `keg evf deps --update`
      *  It should update `@keg-hub/jsutils` to the latest version in the `tap-events-force` `pacakge.json` file
      * Open the taps `package.json` and ensure it was updated to the correct version

## Notes
* Currently this only works for `keg-hub` repos
* Eventually I'll add `keg-cli` repos as well
* When I started this branch, `keg-cli` was not a mono-repo, so it didn't needed
